src/fleet/azext_fleet/tests/latest/test_fleet_scenario.py

```diff
@@ -37,7 +37,7 @@ def generate_ssh_keys(cls):
             key_file.write(TEST_SSH_KEY_PUB)
         return pathname.replace('\\', '\\\\')
 
-    @ResourceGroupPreparer(name_prefix='cli-', random_name_length=8, location='westcentralus')
+    @ResourceGroupPreparer(name_prefix='cli-', random_name_length=8, location='eastus2euap')
     def test_fleet(self):
 
```

Comment:
expecting to see additional test scenarios added in this file, at least for update strategy?

Reply:
We will need to create a new scenario. I discussed with Pankil and think it best to do in next version, given the need for this to merge ASAP.